pytket/binders/circuit/Circuit/add_classical_op.cpp

```diff
@@ -242,15 +242,19 @@ void init_circuit_add_classical_op(
             std::vector<bool> vals(reg.size());
             for (unsigned i = 0; i < reg.size(); i++) {
               args[i] = reg[i];
-              vals[i] = bs[i];
+	      vals[i] = false;
+	      if (i < _TKET_REG_WIDTH) {
+		  vals[i] = bs[i];
+	      }
```

Comment:
```suggestion
	      vals[i] = (i < _TKET_REG_WIDTH) && bs[i];
```
This is simpler I think.

Reply:
Thanks.